tests/dom-shim-tests/dom-shim-test.js

```diff
@@ -1,3 +1,7 @@
+global.navigator = {
+  userAgent: ''
```

Comment:
We're checking the user agent in some of the files now, so, this is needed for those tests not to break.